analyzers/tests/SonarAnalyzer.UnitTest/Helpers/ParameterLoaderTest.cs

```diff
@@ -81,28 +81,32 @@ public void SetParameterValues_WhenGivenSonarLintFileHasIntParameterType_Populat
         public void SetParameterValues_WhenGivenSonarLintFileHasStringParameterType_OnlyOneParameter_PopulatesProperty()
```

Comment:
```suggestion
        public void SonarLintFileHasStringParameterType_OnlyOneParameterIsPopulated()
```

Reply:
I agree that those names are a bit clumsy. I still think that if we modify them we should keep following the [naming convention](https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#naming-your-tests).
- `SetParameterValues` is the name of the method being tested.
- `SonarLintFileWithOneStringParameterType` is the scenario under which it's being tested.
- `PopulatesProperty` is the expected behavior when the scenario is invoked.

So maybe:
```suggestion
        public void SetParameterValues_SonarLintFileWithOneStringParameterType_PopulatesProperty()
```
WDYT? Does it make sense?

Reply:
Yes! 
Let's follow the convention here - I agree with what Tim said - that you can see immediately where is the issue is something fails.